tooling/image-sync/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.13.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.7.0
 	github.com/Azure/azure-sdk-for-go/sdk/containers/azcontainerregistry v0.2.1
-	github.com/containers/image/v5 v5.31.1
+	github.com/containers/image/v5 v5.32.0
```

Comment:
Helped dependabot out by updating `containers/image/v5` instead of the indirect `docker/docker` dependency directly